src/Api/Checkout.php

```diff
@@ -0,0 +1,162 @@
+<?php namespace Lean\Woocommerce\Api;
+
+use Lean\AbstractEndpoint;
+use Epoch2\HttpCodes;
+use Lean\Woocommerce\Utils\ErrorCodes;
+use Lean\Woocommerce\Utils\Hooks;
+
+/**
+ * Class Checkout.
+ *
+ * @package Lean\Woocommerce\Api
+ */
+class Checkout extends AbstractEndpoint
+{
```

Comment:
Yes I mean I don't have a personal preference but I think we should follow the same rule that the methods follows. 

Yes that's right I'm going to see if there an issue on the CI rules.